Makefile

```diff
@@ -0,0 +1,29 @@
+PROJECT := ~/go/src/github.com/kubernauts/tk8
```

Comment:
Can you remove the hard coded paths? Name this as `github.com/kubernauts/tk8`

Reply:
@frezbo , PR for Makefile is already there #9